README.md

```diff
@@ -43,12 +44,11 @@ This will start `cargo-marker`, load [`./marker_lints`](./marker_lints) as a *li
 
 Internal structure of marker and its components is documented in [`docs/internal/`](./docs/internal/).
 
-## License
+---
 
-Copyright (c) 2022-2022 Rust-marker
-
-Rust-marker is distributed under the terms of both the MIT license
-and the Apache License (Version 2.0).
-
-See [LICENSE-APACHE](./LICENSE-APACHE), [LICENSE-MIT](./LICENSE-MIT).
+#### License
 
+<sup>
+Licensed under either of <a href="LICENSE-APACHE">Apache License, Version
+2.0</a> or <a href="LICENSE-MIT">MIT license</a> at your option.
```

Comment:
I don't think this change is quite right. rust, clippy, cargo and rust-analyzer all state `under the terms of both the MIT license and the Apache License (Version 2.0)` I took this text from one of those. Removing the copyright text again make sense.

Reply:
That's... interesting, I would assume they have different meanings, this was copied from serde, since the original didn't sound quite right to me, but if everyone else does it this way, I probably shouldn't change this, thanks for pointing this out

Reply:
Maybe we can take the text from Cargo to make sure, that this is correct. (Slightly adjusted) That would look like this:

---

## License

Marker distributed under the terms of both the MIT license and the Apache License (Version 2.0).

See [LICENSE-APACHE](LICENSE-APACHE) and [LICENSE-MIT](LICENSE-MIT) for details.

---

```
## License

Marker distributed under the terms of both the MIT license and the Apache License (Version 2.0).

See [LICENSE-APACHE](LICENSE-APACHE) and [LICENSE-MIT](LICENSE-MIT) for details.
```

Reply:
Oh, right, I remember now, I did this change because of the copyright notice in the readme, without it, it's practically the same, I think just removing the copyright line should be fine, what do you think?

Reply:
I agree, just removing the copyright line sounds good to me :)